sky/cli.py

```diff
@@ -3151,23 +3151,47 @@ def _output():
             else:
                 name, quantity = accelerator_str, None
 
+        # Case-sensitive initially
         result = service_catalog.list_accelerators(gpus_only=True,
                                                    name_filter=name,
                                                    quantity_filter=quantity,
                                                    region_filter=region,
-                                                   clouds=cloud)
+                                                   clouds=cloud,
+                                                   case_sensitive=True)
+        note_included = False
+
         if len(result) == 0:
-            quantity_str = (f' with requested quantity {quantity}'
-                            if quantity else '')
-            yield f'Resources \'{name}\'{quantity_str} not found. '
-            yield 'Try \'sky show-gpus --all\' '
-            yield 'to show available accelerators.'
-            return
+            result_not_case_sensitive = service_catalog.list_accelerators(gpus_only=True,
+                                                   name_filter=name,
+                                                   quantity_filter=quantity,
+                                                   region_filter=region,
+                                                   clouds=cloud,
+                                                   case_sensitive=False)
+            if len(result_not_case_sensitive) > 0:
+                result = result_not_case_sensitive
+                yield '*NOTE*: be cautious of the capitalization '
+                yield 'of your accelerator selection. '
+                yield 'Please double check that the identified '
+                yield 'accelerator is the one you intended \n'
+                note_included = True
```

Comment:
This may not be a needed hint. For example, in `sky launch --gpus h100` we accept it as is.

Can this fix work by simply adding one line L3160 alone? Seems sufficient.

Reply:
Ah I see. Yep the one line by itself works too. I'll update the branch.